publish_frame: map
robot_frame: base_link
twist_topic: /vel
fix_topic: /fix
num_particles: 1000
publish_rate: 100
#ess_threshold: 50